azuredeploy.json

```diff
@@ -85,21 +96,39 @@
       ],
       "properties": {
         "name": "[parameters('appName')]",
-        "serverFarmId": "[resourceId('Microsoft.Web/serverfarms', parameters('hostingPlanName'))]"
-      },
-      "resources": [
-        {
-          "apiVersion": "2018-02-01",
-          "name": "web",
-          "type": "config",
-          "dependsOn": [
-            "[concat('Microsoft.Web/sites/', parameters('appName'))]"
+        "serverFarmId": "[resourceId('Microsoft.Web/serverfarms', parameters('hostingPlanName'))]",
+        "siteConfig": {
+          "appSettings": [
+            {
+              "name": "DOCKER_REGISTRY_SERVER_URL",
+              "value": "https://index.docker.io"
+            },
+            {
+              "name": "DOCKER_REGISTRY_SERVER_USERNAME",
+              "value": ""
+            },
+            {
+              "name": "DOCKER_REGISTRY_SERVER_PASSWORD",
+              "value": null
+            },
+            {
+              "name": "WEBSITES_ENABLE_APP_SERVICE_STORAGE",
+              "value": "false"
+            },
+            {
+              "name": "DISABLE_SIGNUP_WITHOUT_LINK",
+              "value": "0"
+            },
+            {
+              "name": "LABEL_STUDIO_ONE_CLICK_DEPLOY",
+              "value": "1"
+            }
           ],
-          "properties": {
-            "pythonVersion": "3.7"
-          }
-        }
-      ]
+          "linuxFxVersion": "DOCKER|heartexlabs/label-studio:latest",
```

Comment:
Let's make image tag configurable field like username and password.
Default should be set to `latest`

Reply:
done